<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>page_session_options</class>
 <widget class="QWidget" name="page_session_options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QCheckBox" name="check_session_numlock">
       <property name="text">
        <string>Enable numlock on startup</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="check_session_playloginaudio">
       <property name="text">
        <string>Play chimes on startup</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="check_session_playlogoutaudio">
       <property name="text">
        <string>Play chimes on exit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="check_autoapplinks">
       <property name="statusTip">
        <string>Automatically create/remove desktop symlinks for applications that are installed/removed</string>
       </property>
       <property name="text">
        <string>Manage desktop app links</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="check_watch_app_procs">
       <property name="text">
        <string>Show application crash data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="check_quotes">
       <property name="text">
        <string>Disable Startup Quotes</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <item>
        <widget class="QPushButton" name="push_session_setUserIcon">
         <property name="text">
          <string>Change User Icon</string>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_16">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1" rowspan="2">
    <layout class="QFormLayout" name="formLayout_3">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_24">
       <property name="text">
        <string>Time Format:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="line_session_time"/>
       </item>
       <item>
        <widget class="QToolButton" name="tool_help_time">
         <property name="toolTip">
          <string>View format codes</string>
         </property>
         <property name="text">
          <string notr="true">...</string>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_28">
       <property name="text">
        <string>Sample:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_session_timesample">
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_25">
       <property name="text">
        <string>Date Format:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_21">
       <item>
        <widget class="QLineEdit" name="line_session_date"/>
       </item>
       <item>
        <widget class="QToolButton" name="tool_help_date">
         <property name="toolTip">
          <string>View format codes</string>
         </property>
         <property name="text">
          <string notr="true">...</string>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_29">
       <property name="text">
        <string>Sample:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_session_datesample">
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_27">
       <property name="text">
        <string>Display Format</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="combo_session_datetimeorder"/>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="mywindowmanager">
       <property name="toolTip">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Window Manager</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>100</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>Reset Desktop Settings</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_20">
      <item>
       <spacer name="horizontalSpacer_18">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="push_session_resetSysDefaults">
        <property name="text">
         <string>Restore system defaults</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="push_session_resetLuminaDefaults">
        <property name="text">
         <string>Restore Lumina defaults</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_17">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
